charts/argo-cd/README.md

```diff
@@ -105,6 +105,10 @@ For full list of changes please check ArtifactHub [changelog].
 
 Highlighted versions provide information about additional steps that should be performed by user when upgrading to newer version.
 
+### 5.23.3
```

Comment:
This is not needed.

Reply:
That's what I thought as well when I was making the change but @mbevc1 said to update the changelog.  Did he mean something else?

Reply:
Removed.